spec.html

```diff
@@ -47355,35 +47375,35 @@ <h1>prototype</h1>
 
   <emu-clause id="sec-generator-objects">
     <h1>Generator Objects</h1>
-    <p>A Generator is an instance of a generator function and conforms to both the <i>Iterator</i> and <i>Iterable</i> interfaces.</p>
-    <p>Generator instances directly inherit properties from the object that is the initial value of the *"prototype"* property of the Generator function that created the instance. Generator instances indirectly inherit properties from the Generator Prototype intrinsic, %GeneratorFunction.prototype.prototype%.</p>
+    <p>A Generator is created by calling a generator function and conforms to both the <i>Iterator</i> and <i>Iterable</i> interfaces.</p>
+    <p>Generator instances directly inherit properties from the object that is the initial value of the *"prototype"* property of the generator function that created the instance. Generator instances indirectly inherit properties from %GeneratorPrototype%.</p>
 
-    <emu-clause id="sec-properties-of-generator-prototype">
-      <h1>Properties of the Generator Prototype Object</h1>
-      <p>The <dfn>Generator prototype object</dfn>:</p>
+    <emu-clause id="sec-%generatorprototype%-object" oldids="sec-properties-of-generator-prototype">
+      <h1>The %GeneratorPrototype% Object</h1>
+      <p>The <dfn>%GeneratorPrototype%</dfn> object:</p>
       <ul>
-        <li>is <dfn>%GeneratorFunction.prototype.prototype%</dfn>.</li>
+        <li>is the initial value of <dfn>%GeneratorFunction.prototype.prototype%</dfn>.</li>
```

Comment:
```suggestion
        <li>is <dfn>%GeneratorFunction.prototype.prototype%</dfn>.</li>
```
You can talk about the initial value of an intrinsic object's property (or internal slot), but it doesn't make sense to talk about the initial value of an intrinsic object itself.